src/utilities.h

```diff
@@ -80,4 +80,7 @@ char * PrintTimestamp(void); // TODO remove this function
 unsigned long GetProcessorAndCore(int *chip, int *core);
 void *aligned_buffer_alloc(size_t size, ior_memory_flags type);
 void aligned_buffer_free(void *buf, ior_memory_flags type);
+
+void srand64(uint64_t state);
+uint64_t rand64();
```

Comment:
Should be:

```c
uint64_t rand64(void);
```

Otherwise the following error is thrown:

```
./build/ior/src/utilities.h:85:1: error: function declaration isn't a prototype [-Werror=strict-prototypes]
   85 | uint64_t rand64();
      | ^~~~~~~~
```

Reply:
It doesn't work for me (reads 0 bytes).